test/test.py

```diff
@@ -70,6 +72,10 @@
 1- Beginner
 2- Medium
 3- Advanced
+>>> get_input_standard(test_keyboard_interrupt)
+Traceback (most recent call last):
+...
+SystemExit
```

Comment:
Can we also have `EXIT_MESSAGE` content here?
That's not a big deal, just to make sure everything is tested as much as possible.

Reply:
Rather than that, it's good. If you don't think adding such a thing is necessary you can merge it.

Reply:
Test system behavior is a bit weird here. I think it's because of the `sys.exit()`.